parlai/nn/lr_scheduler.py

```diff
@@ -480,13 +501,13 @@ def __init__(
     def _linear_lr(self, step):
         # this multiplicative factor ensures linear decay rate
         # lr_mult = float(self.max_lr_steps - step - 1) / float(self.max_lr_steps - step)
-        lr_mult = max(0.0, 1.0 - step / self.max_lr_steps)
+        lr_mult = max(0.0, 1e-6 + (1.0 - step / self.max_lr_steps) * (1 - 1e-6))
```

Comment:
what does this do?

Reply:
It makes it so the LR never hits exactly 0 (at most 1e-6 * the peak LR)